app/util/planParamUtil.js

```diff
@@ -196,6 +203,13 @@ export function planQueryNeeded(
         settings.includeBikeSuggestions
       );
 
+    case PLANTYPE.CARTRANSIT:
+      return (
+        transitModes.length > 0 &&
+        settings.includeCarSuggestions &&
+        config.carBoardingModes?.FERRY !== undefined
```

Comment:
This seems wrong. I suggest:  

 settings.includeCarSuggestions && filterTransitModes(...).length > 0

Reply:
I changed this to transitModes.length > 0 && settings.includeCarSuggestions. I didn't realize previously that transitModes is filtered earlier with filterTransitModes.